source/MaterialXContrib/MaterialXNode/MaterialXShadingNodeImpl.cpp

```diff
@@ -8,6 +8,21 @@
 #include <maya/MPxShadingNodeOverride.h>
```

Comment:
Can I ask you to re-merge this file, transferring important changes manually? (see comments below)